common/djangoapps/student/tests/tests.py

```diff
@@ -819,38 +819,6 @@ def test_change_to_honor_if_verified_not_active(self):
         self.assertEqual(enrollment_mode, u'honor')
 
 
-class PaidRegistrationTest(ModuleStoreTestCase):
-    """
-    Tests for paid registration functionality (not verified student), involves shoppingcart
-    """
-    def setUp(self):
-        super(PaidRegistrationTest, self).setUp()
-        # Create course
-        self.course = CourseFactory.create()
-        self.req_factory = RequestFactory()
-        self.user = User.objects.create(username="jack", email="jack@fake.edx.org")
-
-    @unittest.skipUnless(settings.FEATURES.get('ENABLE_SHOPPING_CART'), "Shopping Cart not enabled in settings")
-    def test_change_enrollment_add_to_cart(self):
```

Comment:
Ugh, and now I see that you've removed the conditional in `student/views.py`'s `change_enrollment()` which is responsible for passing the request to the shoppingcart. Ignore me.